src/main/java/io/jenkins/pluginhealth/scoring/service/PluginService.java

```diff
@@ -56,4 +56,21 @@ public Plugin saveOrUpdate(Plugin plugin) {
     public Stream<Plugin> streamAll() {
         return pluginRepository.findAll().stream();
     }
+
+    @Transactional
+    public int getPluginsCount() {
+        return pluginRepository.findAll().size();
+    }
+
+    @Transactional
+    public int getProbeRawData(String probeID) {
+        switch (probeID) {
+            case "up-for-adoption":
+            case "security":
+            case "deprecation":
+                return pluginRepository.getProbeRawData(probeID, "FAILURE");
+            default:
+                return pluginRepository.getProbeRawData(probeID, "SUCCESS");
+        }
```

Comment:
Found out this syntax.

```suggestion
        return switch (probeID) {
            case "up-for-adoption", "security", "deprecation" ->
                pluginRepository.getProbeRawData(probeID, "FAILURE");
            default ->
                pluginRepository.getProbeRawData(probeID, "SUCCESS");
        };
```